src/swagger.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
The instruction for the homework specifies using Swagger js docs for generating API (https://www.npmjs.com/package/swagger-jsdoc)